data/sql/updates/pending_db_world/Scarlet_Group_Patrol.sql

```diff
@@ -0,0 +1,86 @@
+(12873600, 31, 1602.1147, -5781.56, 116.68636, NULL, 0, 0, 0, 100, 0),
+(12873600, 32, 1576.719, -5785.6636, 119.117615, NULL, 0, 0, 0, 100, 0);
+
+DELETE FROM `waypoint_data` WHERE `id` = 12873500;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(12873500, 1, 1773.9238, -5832.1016, 116.31873, NULL, 0, 0, 0, 100, 0),
+(12873500, 2, 1756.1492, -5834.704, 116.430405, NULL, 0, 0, 0, 100, 0);
+
+DELETE FROM `waypoint_data` WHERE `id` = 12873700;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(12873700, 1, 1783.6888, -5806.3467, 114.37271, NULL, 0, 0, 0, 100, 0),
+(12873700, 2, 1765.1584, -5808.4717, 116.48846, NULL, 0, 0, 0, 100, 0);
+
+
+UPDATE `creature` SET `MovementType`= 2 WHERE `guid` = 128736 AND `id1` = 28530;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128705 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128706 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128707 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128708 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128709 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128710 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128711 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128712 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 0, `wander_distance` = 0 WHERE `guid` = 128713 AND `id1` = 28529;
+UPDATE `creature` SET `MovementType`= 2, `position_x`= 1756.1492, `position_y`= -5834.704, `position_z`= 116.430405 WHERE `guid` = 128735 AND `id1` = 28530;
+UPDATE `creature` SET `MovementType`= 2, `position_x`= 1765.1584, `position_y`= -5808.4717, `position_z`= 116.48846 WHERE `guid` = 128737 AND `id1` = 28530;
+
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (128735));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(128735, 12873500, 2404, 0, 0, 0, 0, NULL);
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (128736));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(128736, 12873600, 2404, 0, 0, 0, 0, NULL);
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (128737));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(128737, 12873700, 2404, 0, 0, 0, 0, NULL);
```

Comment:
You could merge the delete from and insert into of creature addon and of the waypoint table